.github/ISSUE_TEMPLATE/4_pre_release.yml

```diff
@@ -1,7 +1,7 @@
 name: 🚀 Pre-release to staging (developers-only)
 description: Issue to plan and log pre-release from master to staging deploy (including staging hotfixes)
 title: "🚀 Pre-release master -> staging_<sprint_name><version>"
-labels: ["t:maintenance", "release"]
+labels: ["t:maintenance", "Release"]

```

Comment:
Q: is it good that ```Release``` becomes the only label with a capital letter? Just wondering.

Reply:
Ok i had changed the label definition to lower case to be consistent